packs/elastic/README.md

```diff
@@ -0,0 +1,170 @@
+# Elasticsearch integration pack
+
+Pack provides many operations helping to manage your Elasticsearch indices. Current functionality is mostly based on [curator](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/).
+
+## Features
+
+Pack provides a set of actions based on curator.
+
+## Curator based actions
+
+Action | Description
+------ | -----------
+**alias** | Add indices to or remove them from aliases.
+**allocation** | Set routing allocation based on tags.
+**bloom** | Disable the bloom filter cache for indices.
+**close** | Close indices.
+**delete.indices** | Delete indices.
+**delete.snapshots** | Delete snapshots.
+**open** | Open indices.
+**optimize** | Optimize indices.
+**replicas** | Set replica count per shard.
+**show.indices** | Show indices.
+**show.snapshots** | Show snapshots.
+**snapshot** | Capture snapshot of indices.
+
+Actions invocation parameters will be described further. But for more detailed description what each action actually does please also refer to the [curator docs](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/), it is more in-depth.
+
+### Common parameters
+
+These parameters include general options such as elasticsearch host, port etc.
+
+Parameter | Description | Default
+------------ | ------------ | ------------
+**host** | Specifies Elasticsearch host to connect to (**required**). | `none`
+**url_prefix** | Specifies Elasticsearch http url prefix. | `/`
+**port** | Specifies port remote Elasticsearch instance is running on. | `9200`
+**use_ssl** | Set to `true` to connect to Elasticsearch through SSL. | `false`
+**http_auth** | Colon separated string specifying HTTP Basic Authentication. | `none`
+**master_only** | Set to `true` enable operation only on elected master. If connected to a node other than master operation will fail. | `false`
```

Comment:
No it doesn't forward anything, just exists with an error. I haven't reworked client connection implementation.